db/migrate/20240111001706_create_likes.rb

```diff
@@ -0,0 +1,14 @@
+class CreateLikes < ActiveRecord::Migration[7.1]
+  def change
+    create_table :likes do |t|
+      t.bigint :UserId
+      t.bigint :PostId
```

Comment:
- [ ] Hello, you're doing great! 🌟 In alignment with Rails conventions, let's ensure we use snake_case naming for the table columns. Specifically, for consistency, modify the column names in your migration to adhere to this convention: